src/ctr_exit.c

```diff
@@ -159,6 +160,14 @@ void do_exit_command()
 	close_fd(&console_socket_fd);
 	close_fd(&attach_pipe_fd);
 
+	/*
+	 * We don't want the exit command to be reaped by the parent conmon
+	 * as that would prevent double-fork from doing its job.
+	 * Unfortunately, that also means that any new subchildren from
+	 * still running processes could also get lost
+	 */
+	prctl(PR_SET_CHILD_SUBREAPER, 0);
```

Comment:
could you check for errors?

Reply:
I don't mind checking, but what should we do if the prctl call fails?
In my opinion we need to run the exit command anyway so at most I can add a warning (nwarnf with errno?), is that OK?

Reply:
hm, perhaps run and skip the wait loop if the prctl failed, but I won't have any way of testing this really (error injection is fun but takes a bit more time than I have short term)

Reply:
I think a warning is enough, at least we have a way to know if we get into this state if we see again the issue you are addressing

Reply:
works for me! added a `nwarn` and force pushed. I've also fixed the issue fix link now the issue has been transferred to #310 (thanks)